userguide/tutorials/aws-cf.adoc

```diff
@@ -119,6 +120,28 @@ Otherwise, you will see that your stack is being created, and its status (shown
 
 If the create succeeds, the status will eventually change to CREATE_COMPLETE (shown in green). This may take a fairly long time.
 
+== Update Configuration
+To update the Kill Bill configuration parameters, select the CloudFormation stack and click on *Update*. The update stack consists of the following 4 steps:
+
+* Step 1 - Update stack: select *Use current template* and click Next
+
+image::https://github.com/vnandwana/killbill-docs/raw/v3/userguide/assets/aws/cf-update-config-step1.JPG[align=center]
+
+* Step 2 - Specify stack details: update the values of any of the parameters and click Next
+
+image::https://github.com/vnandwana/killbill-docs/raw/v3/userguide/assets/aws/cf-update-config-step2.JPG[align=center]
+
+* Step 3 - Configure stack options: no changes are needed on this step
+
+image::https://github.com/vnandwana/killbill-docs/raw/v3/userguide/assets/aws/cf-update-config-step3.JPG[align=center]
+
+* Step 4 - Review: review all the changes made to the parameters, scroll down and check *I acknowledge that AWS CloudFormation might create IAM resources* and click on *Submit*. 
+
+image::https://github.com/vnandwana/killbill-docs/raw/v3/userguide/assets/aws/cf-update-config-step4a.JPG[align=center]
+
+image::https://github.com/vnandwana/killbill-docs/raw/v3/userguide/assets/aws/cf-update-config-step4b.JPG[align=center]
+
```

Comment:
Any particular reason for creating two separate screenshots here?

Reply:
@vnandwana, is it necessary to have separate screenshots? The resultant image looks a bit off: 
<img width="443" alt="image" src="https://user-images.githubusercontent.com/85998496/221086267-25ae87b1-c1ee-4578-897e-f1eb968de91d.png">

Reply:
Looks good now, thanks!